de.dlr.sc.virsat.swtbot.test/META-INF/MANIFEST.MF

```diff
@@ -43,7 +43,20 @@ Require-Bundle: org.eclipse.swt,
  de.dlr.sc.virsat.uiengine.ui,
  de.dlr.sc.virsat.concept.unittest.util,
  de.dlr.sc.virsat.model.extension.budget.mass,
- de.dlr.sc.virsat.model.extension.budget.mass.ui
+ de.dlr.sc.virsat.model.extension.budget.mass.ui,
+ org.hamcrest.library,
+ de.dlr.sc.virsat.model.extension.budget.power,

```

Comment:
Why do we need to add all these dependencies? I don't see them used in the tests

Reply:
Good question. SWTBot seems to start up the VirSat/Eclipse application differently then we do it. Our application is usually based on features. For SWTBot even specifiying the launch configuration the same way, seems to do things differently. If these dependencies are not explicitly added, Virtual Satellite is not executed correctly. 

For example, all concepts will be discovered and displayed in the concept palette. The according images will not be shown because the plugins containing the icons cannot be found/loaded.

Adding the dependencies all images and icons are found as exepcted.